src/Core/OnExceptionDispatchers/OnExceptionExitDispatcher.cs

```diff
@@ -0,0 +1,27 @@
+﻿// <copyright file="OnExceptionExitDispatcher.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.MessageDispatch.Core
+{
+    using System;
+
+    /// <summary>
+    /// Sets up a dispatcher which Exits when an exception is thrown.
```

Comment:
=> something like "A wrapping message dispatcher which kills the application when an exception is thrown from the inner dispatcher."